runtime/src/main/scala/com/soundcloud/twinagle/ClientEndpointBuilder.scala

```diff
@@ -9,7 +9,7 @@ import scalapb.{GeneratedMessage, GeneratedMessageCompanion}
   */
 class ClientEndpointBuilder(
     httpClient: Service[Request, Response],
-    extension: EndpointMetadata => Filter.TypeAgnostic,
+    `extension`: EndpointMetadata => Filter.TypeAgnostic,
```

Comment:
per the comment above: if the generated code doesn't reference the parameter by name, we can also rename it if `extension` is a keyword now